packages/riipen-ui-docs/src/pages.js

```diff
@@ -75,10 +75,22 @@ const pages = [
         name: "Link",
         pathname: "/components/link"
       },
+      {
```

Comment:
Bleh, we should really make this file auto generating. One day :).

Reply:
little tough as we dont need this generated for every component, only the ones that have a purpose. ie, list items dont need their own page as they are displayed in the list page